### ISSUE TITLE
Unsatisfactory Result

### ISSUE THREAD

**Issue by lenacarrier-ai** [leemojiang/D-LUT#3]
I set the same parameters, but get diffrent reslut from the paper fig7. this is my config.yml:

seed: 12213
root_dir: ./Log

model_type: "Energy"
swish: "True"

train_epochs: 15
save_epoch: ~
save_model_flag: True
batch_size: 128
learning_rate: 0.001
sigma: 0.005

style_dir: ./Imgs/style/1.jpg
style_shape:
- 400
- 300
add_uniform: False
uniform_percentage: 0.05

L_steps: 40  # 40
eps: 0.0004  # 0.0002

content_dir: ./Imgs/content
content_shape:
- 300
- 400

lut_dim: 15
lut_sample: 50

model_dir: ~
vis_dim: 20

![Image](https://github.com/user-attachments/assets/488b24cd-3de3-40e8-8e13-462553406b19)

![Image](https://github.com/user-attachments/assets/9daf4df9-0d11-407f-ad72-f2f9bdbc75af)
